build/block-library/post-date/block.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://json.schemastore.org/block.json",
+  "apiVersion": 2,
+  "name": "gutenberg-ant-design/post-date",
+  "version": "0.1.0",
+  "title": "Post Date",
+  "description": "Add the date of this post inside a Query loop.",
+  "textdomain": "gutenberg-ant-design",
+  "usesContext": [
+    "postId",
+    "postType",
+    "queryId"
+  ],
+  "category": "ant-design",
+  "attributes": {
+    "api": {
+      "type": "object"
+    },
+    "settings": {
+      "type": "object"
+    },
+    "visibility": {
+      "type": "array"
+    },
+    "styles": {
+      "type": "object"
+    }
+  },
+  "editorScript": "file:./index.js",
+  "editorStyle": "file:./index.css",
+  "style": "file:./style-index.css",
+  "supports": {
+    "customClassName": false
+  }
+}
```

Comment:
@bmarshall511 there is an EOD missing here. But for some odd reason the build keeps generating file with no last line even though i have tried to add few empty lines.